companion/lib/Service/Satellite.js

```diff
@@ -125,7 +130,10 @@ class ServiceSatellite extends ServiceBase {
 			}
 		}
 
-		const streamColors = params.COLORS !== undefined && !isFalsey(params.COLORS)
+		const streamColors =
+			params.COLORS && ['01', '1', 'true', 'hex', 'rgb'].includes(params.COLORS.toString().toLowerCase())
```

Comment:
I think it would be better to change this so that it either produces one of `'hex' | 'rgb' | null`. that will make for an unambiguous interpretation of what this value means.

It might be worth creating a help function for `parseStringParamWithBooleanFallback(['hex', 'rgb'], 'hex', params.COLOR)` which will try to match the parameter against the list, then fallback to the `isFalsey` check, but always producing a string or null.